source/_data/plugins.yml

```diff
@@ -30,6 +30,11 @@
       link: https://github.com/cypress-io/cypress-webpack-preprocessor
       keywords: [webpack]
 
+    - name: cypress-select-tests
+      description: User space solution for grepping Cypress tests to run
+      link: https://github.com/bahmutov/cypress-select-tests
+      keywords: [browserify]
+
```

Comment:
Do you think this is best in preprocessors? I mean, I know it technically is - but wouldn't this be best under development tools?

Reply:
good point :)